gr-blocks/include/gnuradio/blocks/plateau_detector_fb.h

```diff
@@ -61,6 +61,9 @@ namespace gr {
 	* \param threshold Anything above this value is considered a plateau
 	*/
        static sptr make(int max_len, float threshold=0.9);
+
+       virtual void set_threshold(float threshold) = 0;
+       virtual float get_threshold() = 0;
```

Comment:
Our coding style for getters is to omit the leading get_ and just use a public function name that is the same as the internal variable name.

Reply:
Ah; OK. So just "threshold" to get the value. NP will do for both.